example/01-bootstrap.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
```

Comment:
Please remove this blank line. Docblocks of the file should be placed directly after the PHP tag (all other files have that as well).